app/src/main/res/values/strings.xml

```diff
@@ -5,7 +5,7 @@
     <string name="toast_error_generic">Something bad happened. Couldn\'t open file.</string>
     <string name="toast_error_find_file">Couldn\'t find file. Maybe it doesn\'t exist any longer?</string>
     <string name="toast_error_illegal_file">This doesn\'t seem to be a supported file format.</string>
-    <string name="toast_error_illegal_file_reopen">This doesn\'t seem to be a supported file format. Open it in another app instead.</string>
+    <string name="toast_error_illegal_file_reopen">Unsupported file format. Try opening a document.</string>
```

Comment:
I like how you rephrased it to "unsupported file format", but the second sentence doesn't deliver the correct message anymore. What we'd like to say here is that the app can look for other apps on your phone that DO support that specific file format.